src/modules/ffmpeg/util/av_util.cpp

```diff
@@ -122,47 +124,49 @@ core::pixel_format get_pixel_format(AVPixelFormat pix_fmt)
 core::pixel_format_desc pixel_format_desc(AVPixelFormat pix_fmt, int width, int height, std::vector<int>& data_map)
 {
     // Get linesizes
-    AVPicture dummy_pict;
-    avpicture_fill(&dummy_pict, nullptr, pix_fmt, width, height);
+    int linesizes[4];
+    av_image_fill_linesizes(linesizes, pix_fmt, width);
```

Comment:
You can use function av_image_fill_arrays in place of avpicture_fill
uint8_t* data[4];
int linesizes[4];
av_image_fill_arrays(data, linesizes, nullptr, pix_fmt, width, height, AV_FRAME_ALIGN);

Reply:
Yes, although the data pointers are only used in one of the conditions below, so it seems cleaner to only create them when they're needed.